src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -286,8 +287,37 @@ private bool TryGetTextForContextualKeyword(SyntaxToken token, out string text)
             text = null;
             return false;
         }
+        private static bool TryGetTextForCombinationKeyword(SyntaxToken token, out string text)
+        {
+            // Contextual keywords can appear in any order, and the user could initiate help from either keyword
+            // so to keep the actual checks simple we just check all 4 combinations
```

Comment:
❔ What about `private static protected`? 😨

Reply:
![](https://media.giphy.com/media/jnQYWZ0T4mkhCmkzcn/giphy.gif)

Well the VB implementation will be easy enough...

Reply:
you can just grab teh parent, grab the modifiers off of htat, and just do a .Any for the modifiers you care about :)

Reply:
Yeah, my reaction was more about the horror of the possibilities